### DISCUSSION TITLE
1.0 Preview 1 now available!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1437] [Announcements]
## 1.0 Preview 1 Release of Windows App SDK 

We've just released our first preview of the 1.0 release of the Windows App SDK! This preview is our first release that converges our NuGet packages to one single package, reducing clutter in your projects and making it easier to update to future versions! Additionally, the [AppInstance APIs](https://docs.microsoft.com/windows/apps/windows-app-sdk/applifecycle/applifecycle-instancing) are now available to C# apps! And the [MSAL Library](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-windows-desktop) finally works with WinUI 3 Desktop apps! This is an awesome opportunity to try out what's new and provide feedback.  

If you're building an app with the Windows App SDK for the first time, follow the [getting started instructions here.](https://docs.microsoft.com/windows/apps/project-reunion/get-started-with-project-reunion) 

[Install the C# or C++ extensions](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment) for the 1.0 Preview 1 release. 

Full [release notes here](https://docs.microsoft.com/windows/apps/windows-app-sdk/preview-channel). 

## What's next 

To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://github.com/microsoft/ProjectReunion/blob/main/docs/roadmap.md), which gets updated regularly.  __

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.0-preview1'>1.0.0-preview1</a>.</em>

**Comment by riverar** [comment]
For folks looking for the runtime redistributable package that contains the MSIX files and runtime installers https://aka.ms/windowsappsdk/1.0-preview1/msix-installer

**Comment by ghost1372** [comment]
Every day worse than yesterday

**Comment by shelllet** [comment]
when does `winui3` support unpackaged desktop app? this is what i want to see in the new version.

the other is floating clouds.

**Comment by shelllet** [comment]
not bad , i changed littte code and successfully to upgrade to 1.0-preview from 1.0-exp.

**Comment by edoust** [comment]
The project template included in the linked Visual Studio extension is not compiling, also updating the packages in an existing app does not compile

Is there a specific documentation on how to use this version or a sample app?
